Composer/packages/client/src/recoilModel/dispatchers/publisher.ts

```diff
@@ -31,6 +31,24 @@ const PUBLISH_SUCCESS = 200;
 const PUBLISH_PENDING = 202;
 const PUBLISH_FAILED = 500;
 
+const missingDotnetVersionError = {
+  message: formatMessage('To run this bot, Composer needs .NET Core SDK.'),
+  linkAfterMessage: {
+    text: formatMessage('Learn more.'),
+    url: 'https://docs.microsoft.com/en-us/composer/setup-yarn',
+  },
+  link: {
+    text: formatMessage('Install Microsoft .NET Core SDK'),
+    url: 'https://dotnet.microsoft.com/download/dotnet-core/3.1',
+  },
+};
+
+const checkIfDotnetVersionMissing = (err: any) => {
+  return /(Command failed: dotnet build)| (Command failed: dotnet user-secrets)|(install[\w\r\s\S\t\n]*\.NET Core SDK)/.test(
```

Comment:
I'm not sure I understand that character class. `\s\S` together means "any character at all", so just `install.*\.NET` should work fine.